docs/contribute/pull_request.rst

```diff
@@ -118,3 +118,6 @@ If you want to run a single test:
   rm -rf python/tvm/*.pyc python/tvm/*/*.pyc python/tvm/*/*/*.pyc
 
   TVM_FFI=ctypes python -m pytest -v tests/python/unittest/test_pass_storage_rewrite.py
+
+  #Additionally if you want to run a single test, for example test_all_elemwise inside a file.
```

Comment:
not : space between `#` and `Additionally`

Reply:
` # Additionally if`

Reply:
Ah, thanks - fixed.